scripts/build/mirror.ts

```diff
@@ -89,6 +89,8 @@ export const getMatchingBrowserVersion = (
     );
   }
 
+  let lastRelease;
```

Comment:
To make this a bit clearer:

```suggestion
  let previousReleaseEngine;
```